Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -497,4 +505,20 @@ static u32 ComputeNameSize(const File::FSTEntry& parent_entry)
   return name_size;
 }
 
+static char ASCIILowercaseChar(char c)
+{
+  if (c >= 'A' && c <= 'Z')
+    c += 0x20;
+  return c;
+}
+
+static std::string ASCIILowercaseString(const std::string& str)
+{
+  std::string copy = str;
+  // We can't use tolower, because its behavior depends on the locale,
+  // and we want to be deterministic
+  std::transform(copy.begin(), copy.end(), copy.begin(), ASCIILowercaseChar);
```

Comment:
How do I do that as part of the call to std::transform? Do I have to use a lambda expression? (I'm assuming it isn't possible to just pass the string to tolower.)